app/views/posts/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= stylesheet_link_tag('index') %>
+<div class="container">
+  <div class="user-post">
+    <h3>Post #1 by username</h3>
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aperiam doloremque ex officiis quae veniam.
+      Illum nobis numquam pariatur porro praesentium provident quas repellat. Accusamus deserunt iste laboriosam
+      pariatur perspiciatis sequi.
+    </p>
+  </div>
+  <div>
+    <p>username comment 1</p>
+    <p>username comment 2</p>
+    <p>username comment 3</p>
+    <p>username comment 4</p>
+    <p>username comment 5</p>
+    <p>username comment 6</p>
+  </div>
+</div>
```

Comment:
- You have done well setting up your view file for the `post#show action`, but according to the project requirements, you are to set up a `posts` controller with the show action, please fix this 😄 
**OPTIONAL** - Also, please note that you are not required to have any logic in your view files for this milestone, your view file is allowed to have just a placeholder. Please fix this 👍 👍